lms/static/scripts/frontend_apps/components/test/AutoGradingConfigurator-test.js

```diff
@@ -80,7 +80,7 @@ describe('AutoGradingConfigurator', () => {
         assert.equal(inputs.length, activityCalculation === 'separate' ? 2 : 1);
         assert.equal(
           firstInput.text(),
-          `Annotations${activityCalculation === 'cumulative' ? ' and replies' : ''}Minimum`,
+          `Annotations${activityCalculation === 'cumulative' ? ' and replies' : ''}Goal`,
```

Comment:
The last part of the text is not really important for this particular test, as it changes based on the grading type, which is covered by a different test.

This test only tries to cover that, depending on the activity calculation, the first input has the label "Annotations" or "Annotations and replies".

We could do `startsWith(...)` and ignore the suffix, but `Annotations and replies` would return `true` for `startsWith('Annotations')`, making the test a bit more brittle.

That's why in the end I tested the full text, even though it means potentially breaking it for an unrelated change.